test/node/decimal128_tests.js

```diff
@@ -1216,6 +1217,24 @@ describe('Decimal128', function () {
     expect(new Decimal128('0.0011').toString()).to.equal('0.0011');
     expect(new Decimal128('0.00110').toString()).to.equal('0.00110');
     expect(new Decimal128('-1e400').toString()).to.equal('-1E+400');
-    done();
+  });
+
+  it('throws appropriate error for invalid constructor arguments', () => {
+    const byteLengthErrMsg = 'Decimal128 must take a Buffer of 16 bytes';
+    const constructorArgErrMsg = 'Decimal128 must take a Buffer or string';
+
+    expect(() => new Decimal128(-0)).to.throw(constructorArgErrMsg);
+    expect(() => new Decimal128(-1)).to.throw(constructorArgErrMsg);
+    expect(() => new Decimal128(10)).to.throw(constructorArgErrMsg);
+    expect(() => new Decimal128(1111111111111111)).to.throw(constructorArgErrMsg);
+
```

Comment:
Could we break this test into two parts - one for the tests that test the constructor argument error message and one for the byteLengthErrorMessage?